projects/packages/videopress/src/client/admin/components/admin-page/index.tsx

```diff
@@ -82,9 +86,18 @@ const useDashboardVideos = () => {
 
 		// react to a page param change
 		if ( page !== pageFromSearchParam ) {
-			setVideosQuery( {
-				page: pageFromSearchParam,
-			} );
+			// store changed and not url
+			// update url to match store update
+			if ( page !== tempPage.current ) {
+				tempPage.current = page;
+				searchParams.setParam( 'page', page );
+				searchParams.update();
+			} else {
+				tempPage.current = pageFromSearchParam;
+				setVideosQuery( {
+					page: pageFromSearchParam,
+				} );
+			}
```

Comment:
About this one, we need to be aware of the same thing at https://github.com/Automattic/jetpack/issues/26720 to allow updating the field from the store and getting it reflected in the URL.